packages/esm-patient-test-results-app/src/overview/common-overview.tsx

```diff
@@ -126,14 +142,22 @@ const CommonOverview: React.FC<CommonOverviewProps> = ({
           const allNormalResults = !data.some((result) => abnormalInterpretation.includes(result.interpretation));
           const patientSummaryDashboardData = data.slice(0, DashboardResultsCount);
           return (
-            <Card allNormalResults={allNormalResults} key={uuid}>
+            <Card
+              allNormalResults={allNormalResults}
+              key={uuid}
+              isPatientSummaryDashboard={isPatientSummaryDashboard}
+              isActiveCard={
+                (activeCard && activeCard === uuid) ||
+                (!activeCard && uuid === overviewData[0][overviewData[0].length - 1])
```

Comment:
I don't understand this. There's a default active card? And it's... whatever is the last card? I'm sure I have that wrong